modules/fmc_24vio/ipmi_definition.ini

```diff
@@ -0,0 +1,73 @@
+[Board]
```

Comment:
@thomascobb  where would I find the information needed to populate this properly?

Reply:
@amcdls  should have the info...